packages/website/docs/using-query-suggestions-plugin.md

```diff
@@ -2,7 +2,137 @@
+import algoliasearch from 'algoliasearch/lite';
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'instant_search_demo_query_suggestions',
+});
+
+Learn how to include suggested searches using the Query Suggestions plugin.
+
+:::note
+
+Using this plugin requires an [Algolia](https://www.algolia.com/) application with the [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) feature enabled.
+
+:::
+
+The most common autocomplete UX is one that displays a list of possible queries, or "query suggestions," that your users can select from as they type.  **Query Suggestions help users find queries that are guaranteed to return results.** They help users type less and find what they are looking for faster.
+
+The term “Query Suggestions” refers to the textual suggestions themselves. Query Suggestions are different from search results. Query Suggestions are only suggestions of better queries. When typing “smartphone”, a user may receive a suggestion to pick a more precise query, such as “smartphone apple” or “smartphone iphone xs”, which would retrieve more specific results.
+
+[Algolia](https://www.algolia.com/) provides a [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) feature that generates suggestions [based on your what your users are searching for and the results within your dataset](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/#how-query-suggestions-works). This tutorial explains how to integrate [Algolia Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) into an autocomplete menu using the [`autocomplete-plugin-query-suggestions`](createQuerySuggestionsPlugin) package.
+
+## Prerequisites
+
+This tutorial assumes that you have:
+- a populated [Query Suggestions index](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/how-to/creating-a-query-suggestions-index/js/)
+- existing markup containing an input element where you want to implement the autocomplete dropdown
+- front-end development proficiency with HTML, CSS, and JavaScript
+
+:::note
+
+If you don't have a Query Suggestions index yet, follow the guide on [creating a Query Suggestions index](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/how-to/creating-a-query-suggestions-index/js/).
+
+:::
+
+To follow along with the tutorial, you can start with the markup and style sheets from the [complete example on GitHub](https://github.com/algolia/doc-code-samples/tree/autocomplete-v1/Autocomplete/multi-source). The tutorial includes code snippets of the JavaScript necessary to add Query Suggestions.
```

Comment:
Sure! Since the prereqs include `existing markup containing an input element where you want to implement the autocomplete dropdown` and there is later an explanation about selecting the correct container, I don't think we need to even inline an HTML file.